Dockerfile

```diff
@@ -7,15 +7,15 @@ RUN curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - \
     && curl https://packages.microsoft.com/config/debian/10/prod.list > /etc/apt/sources.list.d/mssql-release.list \
     && apt-get update && ACCEPT_EULA=Y apt-get install -y --no-install-recommends \
         libgeos-c1v5 \
-        python-numpy \
-        python-scipy \
-        python-matplotlib \
-        ipython \
+        python3-numpy \
+        python3-scipy \
+        python3-matplotlib \
+        python3-ipython \

```

Comment:
for some unknown reason I had to do this :-)

Reply:
after our discussion please try one more time with the opriginal packages (it installed python 2 packages, now it install python 3 packages)